src/psyclone/tests/test_files/gocean1p0/driver_test.f90

```diff
@@ -0,0 +1,58 @@
+!-------------------------------------------------------------------------------
```

Comment:
Do we actually need this file - could we use one of the existing ones? If we do need it, then please correct the copyright/license, the name of the program and the comments since the invoke contains only one kernel.

Reply:
Good question. I need this specific test case since it has all variable access modes (read, write, readwrite), and I remember that I couldn't find an existing gocean example with this.

It might be possible that we could replace another test case somewhere that doesn't care about the various access modes, but I am not sure. I don't have a good solution, some of the tests are used often, and it would be quite a bit of work to check if one existing test could be replaced.

For now I've just added this example - I will have to extend this to also use scalars with all three modes.

Reply:
I left you as author in this file (and me as modifier).